tools/wptrunner/wptrunner/wptrunner.py

```diff
@@ -245,15 +245,10 @@ def run_tests(config, test_paths, product, **kwargs):
                     if test_type == "testharness":
                         run_tests = {"testharness": []}
                         for test in test_loader.tests["testharness"]:
-                            if test.testdriver and not executor_cls.supports_testdriver:
+                            if (test.testdriver and not executor_cls.supports_testdriver) or (
+                                test.jsshell and not executor_cls.supports_jsshell):
```

Comment:
This line needs to be intended one more level.

Reply:
That's for clarity, right? It's functional on my system as written

Reply:
That's for PEP8 compliance and to make the linter happy :)

Reply:
I would have been satisfied if it was just to make Robert happy :P